ETT-Backend/Models/LastUpdatedAt.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using Newtonsoft.Json;
+
+namespace ETT_Backend.Models 
+{
+    public class LastUpdatedAt
```

Comment:
I don't think we need this entire file, this can all be moved into the EmployeeResponse